### ISSUE TITLE
As a developer or journalist, I want to obtain a list of the individual data elements with a definition of what that element means in plain english so that I can gain a thorough understanding of what the data is telling me

### ISSUE THREAD

**Issue by jjoteal** [18F/college-choice#169]
Desktop reference comp:
![image](https://cloud.githubusercontent.com/assets/12192706/8753109/ed755b7a-2c86-11e5-8ad1-30116f1bcada.png)
(Please note that a paragraph about the Category of Data might be necessary. See mobile comp for an example of that if needed)

Mobile reference comp:
![image](https://cloud.githubusercontent.com/assets/12192706/8753110/f650bb5e-2c86-11e5-82e7-c352ba5f4d92.png)

**Comment by LisaGee** [comment]
@jjoteal I expect we'll need to evaluate this layout in conjunction with Clare and the text that we do have available.

**Comment by jjoteal** [comment]
@LisaGee, you accepted the new layout for these in https://github.com/18F/college-choice/issues/454, so closing this ticket.
